### PR TITLE
make sure sort order of usernames does not affect evaluation if we need to post another comment

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

**Comment by erwinmombay** [comment]
@googlebot I signed it!

File: routes/owners/api.js [hunk 1]
```diff
@@ -129,9 +129,14 @@ function maybePostComment(prInfo: PullRequestInfo): Promise<*> {
...
-    // The list of the previous comment is not equal to the list
-    // that we currently see generated from the getting all owners.
-    if (!_.isEqual(reviewers, allFileOwnersUsernames)) {
+    // If the list of reviewers from the last bot's comment is different
+    // from the current evaluation of required reviewers then we need to
+    // post the new list of reviewers. (this might be more than the number
+    // of previously required reviewers or less, but in any case we need to
+    // post the list again)
+    if (reviewers.length != allFileOwnersUsernames.length ||
+        _.intersection(reviewers, allFileOwnersUsernames).length !=
```
Comment (jridgewell) [inline routes/owners/api.js]: Now that these are sorted, you should be able to use `_.isEqual` which'll be faster.

Comment (erwinmombay) [inline routes/owners/api.js]: @jridgewell good point. original code was using isEqual

**Comment by erwinmombay** [comment]
@jridgewell added tests. thanks!

**Pull request merged**